<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDIConfig</class>
 <widget class="QDialog" name="DialogDIConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Имена дискр. входов</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Вход 3:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEdit_DI5"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Вход 5:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="lineEdit_DI11"/>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lineEdit_DI12"/>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="lineEdit_DI13"/>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEdit_DI6"/>
   </item>
   <item row="8" column="3" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Вход 11:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Вход 13:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="lineEdit_DI9"/>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="lineEdit_DI10"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="lineEdit_DI3"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Вход 4:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEdit_DI4"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Вход 8:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Вход 14:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="lineEdit_DI14"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Вход 9:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Вход 1:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Вход 7:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Вход 10:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="lineEdit_DI8"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Вход 12:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Вход 6:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="lineEdit_DI7"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="lineEdit_DI2"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Вход 2:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEdit_DI1"/>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDIConfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDIConfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
